nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/java/org/nuxeo/ecm/platform/comment/impl/CommentsMigrator.java

```diff
@@ -128,6 +237,7 @@ protected void migrateComments(CoreSession session, RelationManager relationMana
         if (parent != null && comment != null) {
             comment.putContextData(DISABLE_NOTIFICATION_SERVICE, TRUE); // Remove notifications
             comment.setPropertyValue(COMMENT_PARENT_ID, parent.getId());
+            stripACLs(comment);
```

Comment:
I would say we don't touch previous migration.

Reply:
after discussion with @NourNuxeo this add this modification is needed to avoid any security issue, with @troger we decided to add it, in a case where there no ACLs modification script as done by Thierry Martin under 10.10 and the next release, in the case where the client decide only to migrate from `Relation` to `Property`

Reply:
Still, if the client wants something secured it has to run the next migration. Issue I see there is that we're going to have two different cases for the same migration state which could make the support analysis more difficult for almost nothing as we have a migration to fix it.